nomad/eval_endpoint_test.go

```diff
@@ -1838,3 +1932,99 @@ func TestEvalEndpoint_Reblock(t *testing.T) {
 		t.Fatalf("ReblockEval didn't insert eval into the blocked eval tracker")
 	}
 }
+
+func TestEvalEndpoint_BigStateDelete(t *testing.T) {
```

Comment:
Oops, forgot to delete this! I was using it as a helper to build myself a snapshot for manual testing 😊